### PR TITLE
exon number column and HGVSc column in mutation table

### PR DESCRIPTION
**Pull request by leexgh**
Add a column to show the exon number in mutation table in result view and patient view. The exon number can be 1~3 digits.
image

image

Add another column to show the HGVSc in mutation table in result view and patient view.
image

Fixes: 
#5609 https://github.com/cBioPortal/cbioportal/issues/5609
#2996 https://github.com/cBioPortal/cbioportal/issues/2996

### CODE REVIEW

File: src/shared/components/mutationTable/MutationTable.tsx [hunk 1]
```diff
@@ -493,6 +495,17 @@ export default class MutationTable<P extends IMutationTableProps> extends React.
             tooltip:(<span>Total number of nonsynonymous mutations in the sample</span>),
             align: "right"
         };
+
+        this._columns[MutationTableColumnType.EXON] = {
+            name: "Exon",
+            render:(d:Mutation[])=>(this.props.genomeNexusCache
```
[the remark this answers is not in the document]
Comment (leexgh) [reply src/shared/components/mutationTable/MutationTable.tsx]: updated, thanks! @onursumer

**Comment by jjgao** [comment]
how is it possible to have `110/10`? is it just a mockup?

**Comment by leexgh** [comment]
@jjgao sorry for the confusion, yes it's a mockup. I just want to show an example when the number has three digits. could you tell me which dataset has 3 digits exon? so I can have a screenshot for real data. Or maybe I could change the mockup to a more reasonable number manually?

**Comment by leexgh** [comment]
@jjgao Fixed. It's a screenshot from real data now. Sorry about the issue.

**Comment by jjgao** [comment]
@leexgh it'd be interesting to see one example of patient view page because the numbers of exons are different there.

**Comment by jjgao** [comment]
I just made one :)

image

http://cbioportal-frontend-pr-1975.herokuapp.com/patient?studyId=ucec_tcga_pub&caseId=TCGA-BK-A0CC

I am not sure if aligning around "/" looks good... @inodb @onursumer thoughts?

@leexgh could you enable sorting at least in the results page?

**Comment by sandertan** [comment]
@leexgh @jjgao Nice to see your are working on adding this feature! From what I can understand of this code change, you are retrieving this HGVSc from genome nexus, right? Will you also be adding the possibility to get this from the data? It is valid MAF format (See "HGVSc" in https://docs.gdc.cancer.gov/Data/File_Formats/MAF_Format/)

FYI @pieterlukasse

**Comment by jjgao** [comment]
@sandertan @pieterlukasse  we thought about that. It would require significant db migration effort to add these columns into the existing database. Basically, we have to reannotate all variants in the database. As we are going to rely more on GenomeNexus for real-time annotation, I am not sure if it is worth the effort to add these annotations (or other non-core ones -- ie the ones that are not part of query, e.g. gene symbol or mutation types can be specified in query) into the portal database.

**Comment by inodb** [comment]
I'm planning to merge this n deploy - we can fix other issues in post processing. It's hidden by default anyway

**Pull request merged**